nestjs/src/cross-check/utils/tasksFiltering.ts

```diff
@@ -0,0 +1,20 @@
+import { CourseTask } from '@entities/courseTask';
+
+export const getTaskFilters = () => {
+  const currTimestampUTC = Date.now();
```

Comment:
I am not sure this is good pattern:  since it is in closure there is no guarantee it will be correct value when called by fns below

Reply:
To be honest, this solution seems pretty good to me. Storing the value in a closure allows you to avoid unnecessary time calculations, and also isolates the received time value from external access. But perhaps I missed something, could you please give an example of a problem that may arise here?

Reply:
while i agree about complex calculations, taking current time is not expensive( literally nothing) and u save current time. at the moment when the function is called the time might be changed significantly. there is no problem in you current code. but this is a good place for confusion and future errors